shell/platform/darwin/ios/BUILD.gn

```diff
@@ -271,6 +271,24 @@ copy("copy_dylib") {
   deps = [ ":create_flutter_framework_dylib" ]
 }
 
+copy("copy_asan_runtime_dylib") {
+  visibility = [ ":*" ]
+
+  _libclang_base_path =
+      "//buildtools/mac-x64/clang/lib/clang/13.0.0/lib/darwin/"
```

Comment:
Is there any change we can get this put in a path that doesn't contain the clang version, like `//buildtools/mac-x64/clang/lib/clang/current/lib/darwin/`?  The tool will have to do some regexing for `13.0.0` otherwise.

Reply:
GN should fail at `gn gen` time if the file is not found after a toolchain update. I don't know if the toolchain contains a manifest about where certain resources are present.

Reply:
Oh I was looking in the wrong place in this file.
```
outputs = [ "$root_out_dir/$_dylib_name" ]
```
So that would be `out/ios_debug_unopt/libclang_rt.asan_ios_dynamic.dylib`. LGTM

Reply:
Yup!